tests/test_ancillaries.py

```diff
@@ -192,6 +202,32 @@ def test_af_time():
     assert np.all(np.diff(tt) > 0)
 
 
+# def test_af_mock_with_patch(monkeypatch):
+
+    # def calltracker(func):
+    #     """ Decorator to track how many times a function is called """
+    #     def wrap_mock():
+    #         mock_run = Mock()
+    #         return mock_run
+    #     return wrap_mock
+
+    # old_get_bright = dclab.features.bright.get_bright
+
+    # @calltracker
+    # def wrapper_get_bright(mm):
+    #     old_get_bright(mask=mm["mask"], image=mm["image"], ret_data="avg,sd")
+    # mock_run = Mock()
+    # monkeypatch.setattr(
+    #     'dclab.rtdc_dataset.core.RTDCBase.__getitem__.compute', mock_run)
```

Comment:
I think instead of `mock_run`, you should use `wrapper_get_bright` in the monkeypatch?

Reply:
Yeah I was still messing with this when it was pushed.